install_script.sh.template

```diff
@@ -300,6 +300,17 @@ function remove_rpm_gpg_keys() {
     done
 }
 
+function get_relevant_gpg_keys() {
+    local minorVersion="$1"
+    shift
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏What does `shift` do here?

Reply:
`shift` move forward the argument list by one meaning that in arguments of the function I have `get_relevant_gpg_keys  version gpgkey1 gpgkey2 gpgkey3 gpgkey4` after shift it's going to be as `get_relevant_gpg_keys gpgkey1 gpgkey2 gpgkey3 gpgkey4` and `local gpgKeys=("$@")` will take every remaining arguments inside of a new array `gpgKeys`

Reply:
I need to refresh pages before answering to already answered question :sweat_smile:

Reply:
Thanks!